doc/source/cd/background.md

```diff
@@ -274,8 +274,11 @@ the reconstruction loss, e.g. if MSE is used:
 $\phi, \psi = \text{arg} \min_{\phi, \psi}\, \lVert \mathbf{X}-(\phi \circ \psi)\mathbf{X}\rVert^2$.
 However, untrained (randomly initialised) autoencoders can also be used.
 For an example, a `pytorch` autoencoder can be incorporated into a
-detector by packaging it as a callable function using {func}`~alibi_detect.cd.pytorch.preprocess.preprocess_drift`
-and {func}`~functools.partial`: 
+detector by packaging it as a callable function using
+`preprocess_drift` (from
+[alibi_detect.cd.pytorch.preprocess](../api/alibi_detect.cd.pytorch.preprocess.rst))
+and `partial` (see
+[here](https://docs.python.org/3/library/functools.html#functools.partial)):
```

Comment:
Why the change to the link style here and elsewhere?

Reply:
At a loss as to how this has snuck in. Will revert.